src/main/kotlin/notary/IrohaCommand.kt

```diff
@@ -41,7 +41,23 @@ sealed class IrohaCommand {
         val accountId: String,
         val key: String,
         val value: String
-    ) : IrohaCommand()
+    ) : IrohaCommand() {
+
+        companion object {
+            /**
+             * Takes a binary proto command and creates a model command AddPeer
+             * @param bytes the command represented as byte array
+             */
+            fun fromProto(bytes: ByteArray): CommandSetAccountDetail {
```

Comment:
In my opinion it is better to move `fromProto` to `sidechain.ChainEvent'.
Class `notary/IrohaCommand` represents notary intention to write to Iroha chain while `sidechain.ChainEvent` is representation of Iroha sidechain events.